sql/moz-fx-data-shared-prod/telemetry_derived/experiment_events_live_v1/view.sql

```diff
@@ -0,0 +1,63 @@
+CREATE MATERIALIZED VIEW telemetry_derived.experiment_events_live_v1
+OPTIONS
+  (enable_refresh = TRUE, refresh_interval_minutes = 5)
+AS
+WITH desktop_live AS (
+  SELECT
+    submission_timestamp AS timestamp,
+    event.f2_ AS event_method,
+    event.f3_ AS `type`,
+    event.f4_ AS experiment,
+    event_map_value.value AS branch
+  FROM
+    `moz-fx-data-shared-prod.telemetry_live.event_v4`
+  CROSS JOIN
+    UNNEST(
+      ARRAY_CONCAT(
+        payload.events.parent,
+        payload.events.content,
+        payload.events.dynamic,
+        payload.events.extension,
+        payload.events.gpu
+      )
+    ) AS event
+  CROSS JOIN
+    UNNEST(event.f5_) AS event_map_value
+  WHERE
+    event.f1_ = 'normandy'
+    AND event_map_value.key = 'branch'
+)
+SELECT
+  date(`timestamp`) AS submission_date,
+  `type`,
+  experiment,
+  branch,
+  TIMESTAMP_ADD(
+    TIMESTAMP_TRUNC(`timestamp`, HOUR),
+    -- Aggregates event counts over 5-minute intervals
+    INTERVAL(DIV(EXTRACT(MINUTE FROM `timestamp`), 5) * 5) MINUTE
+  ) AS window_start,
+  TIMESTAMP_ADD(
+    TIMESTAMP_TRUNC(`timestamp`, HOUR),
+    INTERVAL((DIV(EXTRACT(MINUTE FROM `timestamp`), 5) + 1) * 5) MINUTE
+  ) AS window_end,
+  COUNTIF(event_method = 'enroll' OR event_method = 'enrollment') AS enroll_count,
+  COUNTIF(event_method = 'unenroll' OR event_method = 'unenrollment') AS unenroll_count,
+  COUNTIF(event_method = 'graduate') AS graduate_count,
+  COUNTIF(event_method = 'update') AS update_count,
+  COUNTIF(event_method = 'enrollFailed') AS enroll_failed_count,
+  COUNTIF(event_method = 'unenrollFailed') AS unenroll_failed_count,
+  COUNTIF(event_method = 'updateFailed') AS update_failed_count,
+  COUNTIF(event_method = 'disqualification') AS disqualification_count,
+  COUNTIF(event_method = 'expose' OR event_method = 'exposure') AS exposure_count
+FROM
+  desktop_live
+WHERE
+  timestamp > TIMESTAMP('2021-03-01')
```

Comment:
This prevents the materialized view to query data from the beginning of time (which would be expensive)

Reply:
Materialized views do not support `CURRENT_DATE()` at the moment. Also [partition expiration can't be set on materialized views.](https://cloud.google.com/bigquery/docs/materialized-views#partition_expiration)